### ISSUE TITLE
No filter box on stats pages

### ISSUE THREAD

**Issue by zcorpan** [GoogleChrome/chromium-dashboard#583]
https://www.chromestatus.com/metrics/css/popularity
and
https://www.chromestatus.com/metrics/feature/popularity

should have a filter box so one can search for a substring (like "appearance").

**Comment by ebidel** [comment]
Good idea. We've been lazy and have relied on in-browser search for this use case.

**Comment by zcorpan** [comment]
Yeah that works but doesn't give you an overview of related but spread-out counters.
